source/Halibut.Tests/UsageFixture.cs

```diff
@@ -136,7 +137,8 @@ await using (var clientAndService = await clientAndServiceTestCase.CreateTestCas
         }
 
         [Test]
-        [LatestAndPreviousClientAndServiceVersionsTestCases(testNetworkConditions: false)]
```

Comment:
It doesn't make sense to measure progress when testing an old client, since in that case we are actually testing latest to old service. Which is a test we already have.